src/common/logger.cpp

```diff
@@ -0,0 +1,23 @@
+#include "common/logger.h"
+
+namespace biscuit::common {
+
+LoggerConfig::LoggerConfig() {
+#ifndef NDEBUG
+  spdlog::set_level(spdlog::level::debug);
+#endif
+  spdlog::flush_every(std::chrono::seconds(ASYNC_FLUSH_TIME));
+}
+
+const std::shared_ptr<spdlog::logger> Logger::LOG = spdlog::basic_logger_mt("default_logger", LOG_FILE);
```

Comment:
Add support for log rotator. Dumping everything in a single file with 1 sec interval is extremely bad. You can't manage this ever-growing file.

Reply:
yes